src/lib.rs

```diff
@@ -431,6 +431,51 @@ impl OutlineBuilder for DummyOutline {
 #[derive(Clone, Copy, PartialEq, Eq, Debug)]
 pub enum RasterImageFormat {
     PNG,
+
+    /// A monochrome bitmap. The most significant bit of the first byte corresponds to the top-left
```

Comment:
In Rust docs, the first line should be short, since this is a title. Then a blank one and then the rest of the docs.

Like so:

```rust
    /// A monochrome bitmap. 
    ///
    /// The most significant bit of the first byte corresponds to the top-left
    /// pixel, proceeding through succeeding bits moving left to right. The data for each row is
    /// padded to a byte boundary, so the next row begins with the most significant bit of a new
    /// byte. 1 corresponds to black, and 0 to white.
```